prepare for episode release (#145)
rustacean-station/rustacean-station.org: _episodes/071-kyle-machulis.md

---
title: "Buttplug with Kyle Machulis"
date: 2022-05-27T15:30:00Z
file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e071-kyle-machulis.mp3
duration: "1:01:30"
length: "59048856"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Kyle Machulis](https://twitter.com/robjtede), lead developer on [Buttplug](https://buttplug.io/). Buttplug is an open-source standards and software project for controlling intimate hardware, including sex toys, fucking machines, and more.

Reviewer: ```suggestion
Allen Wyma talks with [Kyle Machulis](https://twitter.com/robjtede), lead developer on [Buttplug](https://buttplug.io/). Buttplug is an open-source standards and software project for controlling intimate hardware such as sex toys.
```
Just to tone it down slightly :sweat_smile:
